### DISCUSSION TITLE
Parameters for airodump-ng

### DISCUSSION THREAD

**Discussion by ChillVibesMushroom** [aircrack-ng/aircrack-ng#2540] [General]
Can we run airodump-ng without setting a specific channel I was reading something what I understood was if we don't set a channel in  airodump-ng is that automatic channel hopping do we run the -D parameter to force the connection

**Comment by ZerBea** [comment]
Some background information about this question:
I got this issue report and understand the intent behind this request:
https://github.com/ZerBea/hcxdumptool/issues/259

But hcxdumptool is an interactive tool. It transmit and it expect a response on the same(!) channel.
I can't allow that a tool that change the operating channel is running in parallel on the same interface. Running active monitor mode this will cause hcxdumptool to be a spammer (because it will never get a response if the channel is changed).

While all tools of the aircrack-ng suite are (more or less) designed to work together which each other, hcxdumptool does not "like" sharing an interface. The reason is explained above. The same applies if iw is used to add(!) a monitor interface (monx), but leave the physical interface in managed mode. Also the same applies if a Networkmanager is not stopped, before running hcxdumptool.

**Comment by Mister-X-** [comment]
@ChillVibesMushroom I don't understand. Could you add punctuation in your sentence so that it makes sense.

**Comment by ChillVibesMushroom** [comment]
I figured out what to do it's basically a being prepared situation for the target to reconnect.
